cadence/cadence_types.py

```diff
@@ -295,6 +295,7 @@ def value_for(cls, n: int) -> PendingActivityState:
 class HistoryEventFilterType(IntEnum):
     ALL_EVENT = 0
     CLOSE_EVENT = 1
+    INVALID = 2
```

Comment:
why is this needed? Probably for tests?

Reply:
All the enums on grpc now have a `Invalid` value = 0. I'm adding it to all our instances